cmake/config.cmake

```diff
@@ -241,6 +241,7 @@ if(OMR_OS_WINDOWS)
 endif()
 
 set(OMR_SEPARATE_DEBUG_INFO OFF CACHE BOOL "Maintain debug info in a separate file")
+set(OMR_SKIP_STRIP OFF CACHE BOOL "After debug info is extracted, do NOT strip the shared library")
```

Comment:
We don't want to apply this globally; see https://github.com/eclipse-omr/omr/pull/7732#discussion_r2054695023.